app.py

```diff
@@ -179,3 +179,16 @@ def healthcheck() -> responses.PlainTextResponse:
     if not text:
         text = "OK"
     return responses.PlainTextResponse(text, status_code=status_code)
+
+
+# TODO: The endpoint is not called anywhere. This should be done after Qdrant is updated to the newest version.
+@app.post("/update_embedding_payload")
+def update_embedding_payload(
+    project_id: str, embedding_id:str) -> responses.JSONResponse:
```

Comment:
For me this formatting seems a bit weird, missing whitespace and missing linebreak, do you use black?

Reply:
yes, I had it installed but for some reason it wasn't the default formatter, will change it in this PR